specification/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker App Specification
+
+This section describes all the requirements for docker-app interoperability.
```

Comment:
I wouldn't refer directly to `docker-app` as over time it's hopefully just one client.

Reply:
👍